src/test/java/uk/gov/hmcts/ccd/v2/external/controller/BaseCaseAssignedUserRolesControllerIT.java

```diff
@@ -96,6 +100,9 @@
     @Mock
     protected Authentication authentication;
 
+    @MockBean
```

Comment:
```suggestion
    @Inject
```
And would be useful to add an assertion/test in the `AddCaseAssignedUserRolesControllerIT` to check role category is actually set.